web/js/map-OpenLayers.js

```diff
@@ -614,16 +614,19 @@ $.extend(fixmystreet.utils, {
             zoomToBounds( fixmystreet.markers.getDataExtent() );
         }
 
-        $('#hide_pins_link').click(function(e) {
+        $('#hide_pins_link, .big-hide-pins-link').click(function(e) {
```

Comment:
Is it worth having a `hide-pins-link` class on both links and then using that here/below? &lt;shrug>